bucket/materialize.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.78",
+    "description": "Image-to-material converter for use in video games",
+    "homepage": "http://www.boundingboxsoftware.com/materialize/",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "http://www.boundingboxsoftware.com/materialize/download/Materialize_1.78.zip",
+            "hash": "b0f6e3a905781e85dcd7db2383646bce51b8984ab93d1e3d13be42d5bea8904a"
+        }
+    },
+    "bin": "Materialize.exe",
+    "shortcuts": [["Materialize.exe", "Materialize"]],
+    "suggest": {
+        "vcredist": "extras/vcredist2019"
```

Comment:
Link inside FAQ and download page lead into 2013 version.

Reply:
I'm going to test this before changing the VC version...

Reply:
Update: It does indeed require 2013. 2015/17/19 won't do.